source/layouts/layout.erb

```diff
@@ -2,68 +2,47 @@
+                <image src="/assets/images/govuk-logotype-crown.png" class="govuk-header__logotype-crown-fallback-image"></image>
+              </svg>
+              <span class="govuk-header__logotype-text">
+                GOV.UK
+              </span>
             </span>
-            <span class="header__title">
+            <span class="govuk-header__product-name">
               Product Pages
-              <span class="phase-banner">Beta</span>
+              <strong class="govuk-tag govuk-phase-banner__content__tag">
+                beta
+              </strong>
             </span>
           </a>
         </div>
-
-        <div data-module="navigation">
-          <button type="button" class="header__navigation-toggle js-nav-toggle" aria-controls="navigation" aria-label="Show or hide top level navigation">Menu</button>
-
-          <nav id="navigation" class="header__navigation js-nav" aria-label="Top Level Navigation" data-click-events data-click-category="Header" data-click-action="Navigation link clicked">
-            <ul>
+        <div class="govuk-header__content">
+          <button role="button" class="govuk-header__menu-button js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation">Menu</button>
```

Comment:
I can't see this button anywhere, I suspect a call is missing to [initAll()](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md#option-1-include-javascript) as I can't find a grep match for it

Reply:
Ah good spot, I have also missed these two things "Update CSS class names" and "Update and add data-module attributes" from https://github.com/alphagov/govuk-frontend/releases/tag/v3.0.0 which make the menu work

Reply:
I’ve got this working now